lib/model/rwfolder.go

```diff
@@ -1454,6 +1456,7 @@ func (f *sendReceiveFolder) performFinish(state *sharedPullerState) error {
 			// we have resolved the conflict.
 
 			state.file.Version = state.file.Version.Merge(state.version)
+			f.lastModBy = state.file.ModifiedBy.String()
 			if err = osutil.InWritableDir(f.moveForConflict, state.realName); err != nil {
```

Comment:
It gives me this if I try:

cannot use func literal (type func(string)) as type func(string) error in argument to osutil.InWritableDir

It might be because I changed moveForConflict(string) to moveForConflict(string, string).  But how else will I get the modby in there?